custom_components/huesyncbox/select.py

```
@@ -22,51 +26,52 @@ class HueSyncBoxSelectEntityDescription(SelectEntityDescription):
     select_option_fn: Callable[[aiohuesyncbox.HueSyncBox, str], Coroutine] = None  # type: ignore[assignment]
 
 
-class NO_INPUT:
+class NO_INPUT:  # noqa: N801
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `N801`)

Remove unused `noqa` directive

(RUF100)